PresentScreenings/Controllers/FilmInfoDialogController.cs

```diff
@@ -103,6 +104,9 @@ public override void ViewWillAppear()
             _yCurr = _yMargin + _buttonHeight + _yBetweenViews;   // temp
             _yCurr -= _yBetweenViews;
             CreateCancelButton(ref _yCurr);
+
+            // Make the dialog unresizable.
```

Comment:
Please consider to change this comment into "Disable Resizing", and also at other comparable places.

Reply:
Done.